src/test/java/co/cdap/guides/LogAnalyticsAppTest.java

```diff
@@ -63,11 +63,11 @@ public void test() throws TimeoutException, InterruptedException, IOException {
 
     TimeUnit.SECONDS.sleep(5);
 
-    // Start MapReduce job and wait utmost 3 minutes to finish.
+    // Start MapReduce and wait at most 3 minutes to finish
```

Comment:
Definitely don't want "Start MapReduce". Prefer "Start the MapReduce Program" or at the very least "Start the MapReduce".